### ISSUE TITLE
Media at bottom of page overlaps into the footer

### ISSUE THREAD

**Issue by alexdryden** [omeka/omeka-s#1576]
In the default theme, if a media block is placed at the bottom of a page with nothing below it and a full page above it, then the media overlaps with the footer.

Steps to recreate:

1. Create a site and add a page.
2. Add an item with metadata block (use something with enough metadata fields to fill up a significant portion of the page).
3. Add a new Media Embed block AFTER the item w/ metadata.
4. Keep default settings for the media block.
5. Attach an image to the media block.
6. Save and View.

The default footer, Powered by Omeka, appears next to the image. The solution is to add a transparent line break block after the image. That may suggest a solution in the template.

**Comment by DavidFluck** [comment]
Hey, Alex! I was able to reproduce this, and I think I found a fix. This is a style issue with the default theme. This occurs because the default alignment for an image is "float left", which applies a `float: left` in the CSS. This then messes up the sibling `<footer>` element because the footer hasn't been cleared of previous floats. A `clear: both` on the footer fixes it.

I've submitted a PR to the omeka-s-themes/default repository: https://github.com/omeka-s-themes/default/pull/18
